src/main/java/com/iridium/iridiumskyblock/managers/SchematicManager.java

```diff
@@ -28,7 +28,13 @@
 
     public SchematicManager() {
         File parent = new File(IridiumSkyblock.getInstance().getDataFolder(), "schematics");
-        this.schematicPaster = worldEdit || fawe ? new WorldEdit() : new Schematic();
+        boolean isWorldEditBroken = (worldEdit||fawe)?!WorldEdit.isWorking():false;
+        if (isWorldEditBroken)
```

Comment:
I think this should be changed to
```suggestion
        if (!WorldEdit.isWorking() && (worldEdit||fawe))
```

Reply:
Actually it would need to be the other way around (Otherwise it would launch a class not found in the isWorking if worldedit or fawe isn't there, I'll edit it tho.